gcp_variant_transforms/libs/genomic_region_parser.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Function for parsing genomic region."""
+
+from __future__ import absolute_import
+
+import re
+import sys
+from typing import Tuple  # pylint: disable=unused-import
+
+# Matches to regions formatted as 'chr12:10,000-20,000'.
+_REGION_LITERAL_REGEXP = re.compile(r'^(\S+):([0-9,]+)-([0-9,]+)$')
+_DEFAULT_START_POSITION = 0
+_DEFAULT_END_POSITION = sys.maxint
```

Comment:
nit: please use `sys.maxsize` as `maxint` isn't available in python3.

Reply:
Thanks! Glad to know.